pages/bib/[id]/index.tsx

```diff
@@ -142,9 +144,13 @@ export default function BibPage({
         setBib(new Bib(discoveryBibResult))
 
         setItemsLoading(false)
-        setTimeout(() => {
-          itemTableHeadingRef.current?.focus()
-        }, FOCUS_TIMEOUT)
+
+        // TODO: This is a workaround to prevent the Displaying text from receiving focus when filters are controlled via a checkbox
+        // This is an accessibility issue that should be addressed when the dynamic refresh is replaced with a form and apply button
+        if (!refreshedViaCheckbox)
+          setTimeout(() => {
+            itemTableHeadingRef.current?.focus()
+          }, FOCUS_TIMEOUT)
       } else {
         console.log(response)
```

Comment:
does this `console.log` still want to be here?

Reply:
nah, gonna scrap it